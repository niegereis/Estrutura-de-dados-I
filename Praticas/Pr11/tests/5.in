5
10 7 3 4 9 3 2 1 4 1 8
Alice 1217076 28 97.8
Miguel 2211819 68 50.3
Sophia 1426873 86 77.1
Arthur 2914416 81 118.7
Helena 2715645 68 77.5
