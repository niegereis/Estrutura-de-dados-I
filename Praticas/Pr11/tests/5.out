[2914416] Arthur - 81 (118.7)
[      0]  - 0 (0.0)
[1217076] Alice - 28 (97.8)
[2715645] Helena - 68 (77.5)
[1426873] Sophia - 86 (77.1)
